sympy/integrals/tests/test_integrals.py

```diff
@@ -1704,3 +1704,8 @@ def test_issue_4231():
 def test_issue_17841():
     f = diff(1/(x**2+x+I), x)
     assert integrate(f, x) == 1/(x**2 + x + I)
+
+def test_issue_20831():
+    x = symbols('x',positive=True)
```

Comment:
Is the positive=True assumption necessary for the test to pass?

Reply:
I am making change so that the condition is checked for x in the integrand limits. It is not necessary for the test case to pass